File: src/main/java/net/hectus/neobb/turn/default_game/flower/TBlueOrchid.java
package net.hectus.neobb.turn.default_game.flower;

import net.hectus.neobb.player.NeoPlayer;
import net.hectus.neobb.util.Modifiers;
import org.bukkit.block.Block;

public class TBlueOrchid extends FlowerTurn {
    public TBlueOrchid(NeoPlayer player) { super(player); }
    public TBlueOrchid(Block data, NeoPlayer player) { super(data, player); }

    @Override
    public int cost() {
        return 6;
    }

    @Override
    public void apply() {
        player.addModifier(Modifiers.P_DEFAULT_100P_WARP);
    }
}

File: src/main/kotlin/net/hectus/neobb/modes/turn/default_game/flower/TBlueOrchid.kt
package net.hectus.neobb.modes.turn.default_game.flower

import com.marcpg.libpg.storing.Cord
import net.hectus.neobb.player.NeoPlayer
import net.hectus.neobb.util.Modifiers
import org.bukkit.block.Block

class TBlueOrchid(data: Block?, cord: Cord?, player: NeoPlayer?) : FlowerTurn(data, cord, player) {
    override val cost: Int = 6

    override fun apply() {
        player!!.addModifier(Modifiers.Player.Default.ALWAYS_WARP)
    }
}
